docs/source/checkpoints/dist_ckpt.rst

```diff
@@ -0,0 +1,392 @@
+Distributed checkpoints
+=======================
+
+This guide provides details about the distributed checkpoints format from Megatron-Core.
+
+
+Introduction
+------------
+
+Model parallel training requires parallelism-aware checkpointing.
+Megatron-Core provides a checkpointing library capable of handling all types of parallelisms used in LLMs training.
```

Comment:
Megatron Core provides a checkpointing library capable of handling all types of parallelisms used in LLM training.